analyzers/utils/processors/unix.go

```diff
@@ -0,0 +1,84 @@
+package processors
```

Comment:
Can the package be outside `utils`. Then we can directly access `gopath/analyzers/processors/<processor`. A processor is essential for an analyzer to function, it's not merely a helper.